src/main/scala/edu/ie3/simona/model/participant/evcs/EvModelWrapper.scala

```diff
@@ -35,7 +35,7 @@ final case class EvModelWrapper(
   def uuid: UUID = original.getUuid
   def id: String = original.getId
 
-  lazy val pRatedAc: Power = original.getPRatedAC.toSquants
+  lazy val sRatedAc: Power = original.getSRatedAC.toSquants
```

Comment:
I would introduce sRatedAc but also keep pRatedAc and use cosPhi to calculate it (yes it should be the same value since cosPhi should be equal to 1.0).

At least in my understanding the reason for this change was to be aligned over all models.